deploy/kubernetes/webhook-example/rbac-snapshot-webhook.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   name: snapshot-webhook-runner
 rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
-    resources: ["volumesnapshotclasses"]
+    resources: ["volumesnapshotclasses", "volumesnapshots", "volumesnapshotcontents"]
```

Comment:
These additional RBAC rules should not be need here.

Reply:
The webhook will validate these 3 resources: `["volumesnapshotclasses", "volumesnapshots", "volumesnapshotcontents"]`, then how to get the access if not needed here?

Reply:
The webhook does not do a Get or List of "VolumeSnapshots" or "VolumeSnapshotContents".  It works differently from other regular controllers.

"VolumeSnapshotClasses" is different.  There's logic added recently to check it.